### ISSUE TITLE
svgEditorReady event is not triggered in an iframe

### ISSUE THREAD

**Issue by iuyiuy** [SVG-Edit/svgedit#93]
The svgEditorReady event is triggered by referencing the parent window with window.opener, which works when svg-edit is initialized with window.open and not in an iframe.

A temporary fix is to add the event from your config file or in an extension:

``` javascript
svgEditor.ready(function(){
    var svgEditorReadyEvent,
        w = window.parent;
    if (w) {
        try {
            svgEditorReadyEvent = w.document.createEvent('Event');
            svgEditorReadyEvent.initEvent('svgEditorReady', true, true);
            w.document.documentElement.dispatchEvent(svgEditorReadyEvent);
        }
        catch(e) {}
    }
});
```

**Comment by iuyiuy** [comment]
I've created a new pull request for this, which should fix this issue.

**Comment by iuyiuy** [comment]
This was solved in pull request #127, which has been merged. This issue can be closed.
